VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -2851,6 +2851,17 @@ public void Get_EmergingIssues()
             Assert.Equal("AAAAAAAAAAAAAAAAAAAA", ((DeathRecord)XMLRecords[0]).EmergingIssue20);
         }
 
+        [Fact]
+        public void CheckConnectathonRecord1()
+        {
+            DeathRecord dr1 = VRDR.Connectathon.FideliaAlsup();
+            IJEMortality ije = new IJEMortality(dr1, false); // Don't validate since we don't care about most fields
```

Comment:
Aren't the connectathon records intended to be complete records, and so shouldn't they validate? What validation failures show up?

Reply:
I'm just trying to create a blank IJEMortality, and the constructor where you pass in a blank string doesn't work.  This is a workaround:  create an empty DeathRecord, and use the IJEMortality constructor that takes a DeathRecord to create a blank record.   Can't validate because DSTATE is blank, and that is a validation error.   I corrected the comment and made all three cases consistent wrt how the IJEMOrtality record is created.